5_People/career.md

```diff
@@ -26,11 +26,12 @@ This is the Salary Formula:
 
 #### Base Salary
 
-Niteo Base Salary is calculated by taking the [*Salary.com salary base for the role in United States*](https://www.salary.com/research) and converting it to EUR. 
+Niteo Base Salary is calculated by taking the [*Salary.com salary base for the role in United States*](https://www.salary.com/research) and converting it to EUR.
 
 To avoid day-to-day currency fluctuations we take [the 10-year exchange rate average](https://www.ofx.com/en-au/forex-news/historical-exchange-rates/yearly-average-rates/) to convert the Salary.com amounts from USD to EUR, Niteo's primary currency. The exchange rate for 2020 is _1.20_.
 
-Base salaries get calculated once a year, in January, during our [bi-annual IRL](https://blog.niteo.co/irls/). 
+Base salaries get calculated once a year, in January, during our [bi-annual IRL](https://blog.niteo.co/irls/).<br>
+If the new calculation would result in a decreased salary for a Nitean, we instead pin that salary until the next year.
```

Comment:
It's a bit hard to find what was added in this PR because of all the removed whitespace, but here is the added note about pinning a Nitean's salary.